chart/compass/charts/gateway/templates/policy.yaml

```diff
@@ -0,0 +1,12 @@
+# Required because Prometheus Operator doesn't have Istio Sidecar
+apiVersion: security.istio.io/v1beta1
+kind: PeerAuthentication
```

Comment:
Why is this resource needed?

Reply:
It is required, because the prometheus server tries to call the metrics endpoint, but the prometheus server does not have a sidecar. This peerauthentication allows the prometheus to call the metrics endpoint without providing a certificate to the sidecar.